ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml

```diff
@@ -58,7 +58,9 @@ tests:
 - artifact_dir: /tmp/artifacts
   as: e2e-aws-operator
   commands: |
+    openshift-tests run-resourcewatch &
     make test-e2e JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
+    tar -cf ${ARTIFACT_DIR}/rw.tar -C /repository
```

Comment:
Are you tarring while the writer is still running? You need to stop it first as was suggested in the Slack thread.

Reply:
I was wondering about that, the only way I can think to stop the controller is by killing the process because it doesn't really have a "stop" condition, so we can't just wait for it to end. After the e2e test runs, there shouldn't be any more changes to the configs right?
@mfojtik @sanchezl